extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -334,7 +334,11 @@ async function runRemoteQueriesApiRequest(
     void showAndLogInformationMessage(popupMessage, { fullMessage: logMessage });
     return response.data.workflow_run_id;
   } catch (error) {
-    void showAndLogErrorMessage(getErrorMessage(error));
+    if ((error as OctokitResponse<QueriesResponse, number>).status === 404) {
+      void showAndLogErrorMessage(`Controller repository was not found. Please make sure it\'s a valid repo name.${eol}`);
```

Comment:
How come we're adding a new line in the end?

Reply:
To make the error message easier to spot, like so:

![Screenshot 2022-06-22 at 14 26 13](https://user-images.githubusercontent.com/1354439/175040384-9ec0eaec-b629-4410-9fa2-ee5f7631e3f2.png)